modin/tests/pandas/dataframe/test_join_sort.py

```diff
@@ -1012,3 +1012,39 @@ def test_compare(align_axis, keep_shape, keep_equal):
     modin_result = modin_series2.compare(modin_series1, **kwargs)
     pandas_result = pandas_series2.compare(pandas_series1, **kwargs)
     assert to_pandas(modin_result).equals(pandas_result)
+
+
+def test_df_value_counts():
+    df = pd.DataFrame(
+        [[4, 1, 3, 2], [2, 5, 6, 5], [4, 3, 3, 5]], columns=["a", "b", "c", "d"]
+    )
+
+    result_descending = df["a"].value_counts()
+    expected_descending = df._to_pandas()["a"].value_counts()
+    df_equals(result_descending, expected_descending)
```

Comment:
When writing tests, we generally prefer to construct the modin object from a native pandas object rather than calling `_to_pandas()` on a modin object, which could incur performance penalties from serialization. We also have some test helper functions that do that for you, like so:

```suggestion
    eval_general(
        *create_test_dfs([[4, 1, 3, 2], [2, 5, 6, 5], [4, 3, 3, 5]], columns=["a", "b", "c", "d"]),
        lambda df: df["a"].value_counts(),
    )
```